app/rules/timeline-rules-priority.js

```diff
@@ -5,13 +5,18 @@ function atLeastOneJob(jobs) {
   return !!jobs && jobs.length > 0;
 }
 
+function anyJobYoungerThan21days(jobList) {
+  const time21DaysAgo = moment().subtract(21, 'days');
+
+  const jobYoungerThan21days = jobList
+    .find(it => moment(it.statusDate).isAfter(time21DaysAgo, 'day'));
+
+  return jobYoungerThan21days;
+}
+
 function allDatesOlderThen21days(jobList) {
   if (atLeastOneJob(jobList)) {
```

Comment:
Could you move atLeastOneJob into anyJobYouungerThan21Days. The only place the latter funciton is used, atLeastOneJob is sued just before